pinot-core/src/main/java/org/apache/pinot/core/segment/processing/aggregator/PercentileKLLSketchAggregator.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.core.segment.processing.aggregator;
+
+import org.apache.datasketches.common.SketchesArgumentException;
+import org.apache.datasketches.kll.KllDoublesSketch;
+import org.apache.pinot.core.common.ObjectSerDeUtils;
+
+/**
+ * Class to merge KLL doubles sketch for minion merge/rollup tasks.
+ */
+public class PercentileKLLSketchAggregator implements ValueAggregator {
+
+  protected static final int DEFAULT_K_VALUE = 200;
+
+  /**
+   * Given two kll doubles sketches, return the aggregated kll doubles sketches
+   * @return aggregated sketch given two kll doubles sketches
+   */
+  @Override
+  public Object aggregate(Object value1, Object value2) {
```

Comment:
This interface method was updated in https://github.com/apache/pinot/pull/14373. You'll need to rebase your changes (check the compilation failures in CI - https://github.com/apache/pinot/actions/runs/12464118111?pr=14702)

Reply:
Done